packages/replay-internal/src/replay.ts

```diff
@@ -1237,7 +1244,13 @@ export class ReplayContainer implements ReplayContainerInterface {
       if (tooShort) {
         this._debouncedFlush();
       }
-      return;
+
+      // XXX: disregard durations for buffer mode for debug purposes
+      if (this.recordingMode !== 'buffer') {
+        return;
```

Comment:
We only return (and not continue flushing) if in session mode and replay is too long or too short, otherwise setTag and continue flushing (for debugging purposes).